### PR TITLE
README file for entries-api

### PR DESCRIPTION
**Pull request by inexorabletash**

### CODE REVIEW

File: entries-api/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Tests for the [Files and Directory Entries API](https://github.com/wicg/entries-api)
+
+* [Explainer](https://github.com/WICG/entries-api/blob/gh-pages/EXPLAINER.md)
+* [Draft Specification](https://wicg.github.io/entries-api/)
+
+Unfortunately, most of the tests are **manual** and require drag-and-drop of test data.
+They can be run via [w3c-test.org](http://w3c-test.org/entries-api/), but a local
+clone of the repo for access to the `entries-api/support` files.
```
Comment (aliams) [inline entries-api/README.md]: ...local clone of the repo *is required* for access...?

**Review by aliams — CHANGES_REQUESTED** [review]

**Review by aliams — APPROVED** [review]

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-13 18:35:23
Finished: 2018-03-13 18:42:31





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25876)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/352982005)

File: entries-api/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Tests for the [Files and Directory Entries API](https://github.com/wicg/entries-api)
+
+* [Explainer](https://github.com/WICG/entries-api/blob/gh-pages/EXPLAINER.md)
+* [Draft Specification](https://wicg.github.io/entries-api/)
+
+Unfortunately, most of the tests are **manual** and require drag-and-drop of test data.
+They can be run via [w3c-test.org](http://w3c-test.org/entries-api/), but a local
+clone of the repo is required for access to the `entries-api/support` files.
```
Comment (mkruisselbrink) [inline entries-api/README.md]: It wasn't obvious to me until looking at the entries-api/support directory why you would need access to those files. Maybe explicitly mention that the test data it is talking about lives in that directory?

**Comment by inexorabletash** [comment]
Thanks - how's this rewording?

**Review by mkruisselbrink — APPROVED** [review]
thanks!

**Pull request merged**